app/views/bookings/new.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class='container'>
+  <h4>NEW BOOKING REQUEST</h4>
+  <div class='container'>
+    <%= simple_form_for(@booking) do |f| %>
```

Comment:
you have two containers i am not sure the second one is useful?

Reply:
I always contain the page, and the second container is for the form. 
It is useful when styling.
Feedback received on team concensus. Removing.